docs/source/top_level_elements/subject.rst

```diff
@@ -0,0 +1,497 @@
+
+Approximately 12,000 records have at least one subject without a URI. Note that this number is hard to pin down due to
+inconsistencies with where @valueURI is placed on subject nodes. We'll need to treat any of these subjects that aren't able
+to be reconciled as string values. For the postcard collection, the use of dots (Database of the Smokies) as the authority
+makes it impossible to include a URI presently. Other collections with string values that could be
+revisited are: Charlie Daniel Cartoon Collection, Ed Gamble Cartoon Collection, Football Programs, Insurance Company of
+North America Records, the American Civil War Collection, Ramsey Family Papers, Tennessee Documentary History,
+and Volunteer Voices.
+
+Here's an `example record - gamble:123 <https://digital.lib.utk.edu/collections/islandora/object/gamble%3A123/datastream/MODS/view>`_.
+
+.. code-block:: xml
+
+    <subject>
+        <topic>Environmentalism</topic>
+    </subject>
+    <subject>
+        <topic>Factory and trade waste--Environmental aspects</topic>
+    </subject>
+    <subject>
+        <topic>Pollution</topic>
+    </subject>
+    <subject>
+        <topic>Knight</topic>
+    </subject>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:subject "Environmentalism" ;
+        dcterms:subject "Factory and trade waste--Environmental aspects" ;
+        dcterms:subject "Pollution" ;
+        dcterms:subject "Knight" .
+
+Topical subjects with URIs
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Note the variations in where @valueURI is placed in the Xpaths listed below.
+
+    mods:subject[@valueURI]/mods:topic
+    mods:subject/mods:topic[@valueURI]
+
+`acwiley:280 as an example of mods:subject[@valueURI]/mods:topic <https://digital.lib.utk.edu/collections/islandora/object/acwiley%3A280/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <subject authority="lcsh" valueURI="http://id.loc.gov/authorities/subjects/sh85147554">
+        <topic>Women in art</topic>
+    </subject>
+    <subject authority="lcsh" valueURI="http://id.loc.gov/authorities/subjects/sh85147447">
+        <topic>Women artists</topic>
+    </subject>
+    <subject authority="tgm" valueURI="http://id.loc.gov/vocabulary/graphicMaterials/tgm008085">
+        <topic>Portraits</topic>
+    </subject>
+
+`cdf:5384 as an example of mods:subject
+/mods:topic[@valueURI] <https://digital.lib.utk.edu/collections/islandora/object/cdf%3A5384/datastream/MODS/view>`_
+
+.. code-block:: xml
+
+    <subject>
+        <geographic valueURI="http://id.loc.gov/vocabulary/geographicAreas/n-us">United States</geographic>
+    </subject>
+    <subject>
+        <topic valueURI="http://id.loc.gov/authorities/subjects/sh85023396">Child welfare</topic>
+    </subject>
+
+.. code-block:: turtle
+
+    @prefix dcterms: <http://purl.org/dc/terms/> .
+
+    <https://example.org/objects/1> dcterms:subject "http://id.loc.gov/authorities/subjects/sh85023396" .
```

Comment:
I don't care, but just pointing out that we only model the second xml is example as :turtle: .  It doesn't matter in my opinion, but thought I'd add.

Reply:
Yes, since the other value is geographic, I thought it might be confusing to include it in the turtle. Probably it would've been best to simply the example and just not include the geographic subject in the given XML. I'll make that change.